src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,92 @@
+import java.util.*;
 public class StringManipulation implements StringManipulationInterface {
+    private String pString;
 
     @Override
     public String getString() {
-        return null;
+        return pString;
     }
 
     @Override
     public void setString(String string) {
+        this.pString = string;
     }
 
     @Override
     public int count() {
-        return 0;
+        if(pString == null || pString.isEmpty()){
+            return 0;
+        }
+
+        String[] words = pString.split("\\s+"); //split string to words using space
+        return words.length;
     }
 
     @Override
     public String removeNthCharacter(int n, boolean maintainSpacing) {
-        return null;
+        if(n <= 0){
+            throw new IllegalArgumentException("Input n cannot be 0 or negative");
+        }
+
+        String theString = getString();
+        StringBuilder sb = new StringBuilder(theString);
```

Comment:
Good idea using a StringBuilder! I hadn't thought of that in my implementation.

Reply:
But I forgot to add a exception when n is larger than the length of string lol